test/ome-files/tiff.cpp

```diff
@@ -153,7 +164,7 @@ namespace
         idx[ome::files::DIM_CHANNEL] = idx[ome::files::DIM_MODULO_Z] =
         idx[ome::files::DIM_MODULO_T] = idx[ome::files::DIM_MODULO_C] = 0;
 
-      const char * const shades[] = {" ", "░", "▒", "▓", "█"};
+      const std::array<const char *,5> shades{{" ", "░", "▒", "▓", "█"}};
```

Comment:
Note https://msdn.microsoft.com/en-us/library/mt708821.aspx for MSVC.  Its support has historically been quite bad, but VS2015 and above will allow explicit charset selection.  Before that it requires a BOM (for UTF-8!) so currently our VS2013 build simply treats our input as raw bytes in the current system codepage I think.  Not much we can do to address the defects in this platform other than (1) stick with ASCII or (2) ignore it.  Right now we're mainly (2).  The only disadvantage is bad display of Unicode in the console, but you get that with Unicode and Java *anyway* since that's also still stuck with 8-bit codepages!